examples/darwin-framework-tool/commands/common/CHIPCommandBridge.mm

```diff
@@ -134,6 +136,26 @@
 
 MTRDeviceController * CHIPCommandBridge::GetCommissioner(const char * identity) { return mControllers[identity]; }
 
+void CHIPCommandBridge::RestartCommissioners()
+{
+    for (auto & pair : mControllers) {
+        [pair.second shutdown];
+    }
+
+    auto factory = [MTRControllerFactory sharedInstance];
+    NSData * ipk = [gNocSigner getIPK];
+
+    constexpr const char * identities[] = { kIdentityAlpha };
```

Comment:
Again, why is this only restarting alpha?

Reply:
> Again, why is this only restarting alpha?

See https://github.com/project-chip/connectedhomeip/pull/22268#discussion_r959255191
Basically this was not expected to be part of this PR.